WebUI/Pages/GameDetails.razor

```diff
@@ -82,8 +118,18 @@ else
         await wishedItemService.AddToWishlist(game, user.Id);
     }
 
+    private async void SubmitReview() 
+    {
+        await reviewService.AddReview(review);
+    }
+
     private void Download(Game game)
     {
         // TODO: donwload the game
```

Comment:
Make sure to pull the latest changes from master, since there are gonna be merge conflicts.